packages/web-forms/src/assets/css/style.scss

```diff
@@ -38,8 +41,6 @@
 
 	--odk-border-color: var(--p-surface-300);
 
-	--odk-icon-size: 1.5rem;
```

Comment:
This is unlikely to be customized in theming. The primary reason for removing it and letting the IconSVG component handle it is that the SVG's path, which contains the image, is resized using CSS transform.